docs/values.md

```diff
@@ -283,6 +283,10 @@ It might not be desirable to have all the nodes of a kubernetes cluster to parti
 
 By default, AKO prints all the logs to stdout. Instead, persistentVolumeClaim(PVC) can be used for publishing logs of AKO pod to a file in PVC. To use this, the user has to create a PVC (and a persistent volume, if required) and specify the name of the PVC as the value of persistentVolumeClaim.
 
+### securityContext
```

Comment:
@swathinsankaran: `securityContext` applies to individual container.  For AKO, do we really want to set individual container specific security contexts? Can we apply same set of context at POD level?  Should we have separate field?

Reply:
I agree we have only a single container. But some options are unique to`securityContext`.

Reply:
I checked the options which may cause issues in AKO if we set `securityContext`.

 `runAsNonRoot: true` won't work as we are not providing any user during image creation and it is taking the default user which is `root`.

Tried the following settings and it is working.
```
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
      readOnlyRootFilesystem: false
      seccompProfile:
        type: RuntimeDefault
```

Reply:
As discussed, I have changed the binary location to `/usr/bin` so that setting `runAsNonRoot: true` won't return an error.